reference/docs-conceptual/PowerShell-Support-Lifecycle.md

```diff
@@ -24,13 +23,30 @@ guarantee there that the community will address or resolve your issue in a timel
 have a problem that requires immediate attention, you should use the traditional, paid support
 options.
 
-## Lifecycle of PowerShell Core
+## LifeCycle of PowerShell 7
+
+With the release of PowerShell 7, support has moved from the Microsoft Modern Lifecycle Policy to follow [.NET Core's support lifecycle][Long-Term].
+In this servicing approach, customers can choose Long Term Support (LTS) releases or Current
+releases. PowerShell 7.0 is an LTS release. Support ends with the support of .NET Core 3.1.
+The next LTS release follows the next .NET core LTS release. See the [PowerShell releases end of life table](#powerShell-releases-end-of-life)
+for current ending support dates. LTS release updates only contain critical, compatible updates and
+fixes.
```

Comment:
I *think* this is what you meant by compatible? I stumbled a little on it, so just clarifying. Feel free to wordsmith further

```suggestion
for current ending support dates. LTS release updates only contain critical security and servicing updates and
fixes that are designed to avoid or minimize impact to existing workloads.
```

Reply:
Works for me. I tried to translate the .NET terminoligy and meaning so if you think this is more accurate, definitely switching it out.